internal/file/embedded/supported_formats.json

```diff
@@ -0,0 +1,272 @@
+[
```

Comment:
this should be .gitignored and updated through an action before building images, so that it is automated